src/ui/pages/database-detail-pitr.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  fetchDatabase,
+  pitrRestoreDatabase,
+  selectDatabaseById,
+} from "@app/deploy";
+import { useDispatch, useLoader, useQuery, useSelector } from "@app/react";
+import { environmentDatabasesUrl } from "@app/routes";
+import { DateTime } from "luxon";
+import { useEffect, useState } from "react";
+import { useNavigate, useParams } from "react-router";
+import { useLoaderSuccess } from "starfx/react";
+import {
+  Button,
+  DatabaseNameInput,
+  FormGroup,
+  Group,
+  Input,
+  Radio,
+  RadioGroup,
+} from "../shared";
+
+export const DatabasePitrPage = () => {
+  const { id = "" } = useParams();
+  useQuery(fetchDatabase({ id }));
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const database = useSelector((s) => selectDatabaseById(s, { id }));
+  const [handle, setHandle] = useState("");
+  const [recoveryTargetType, setRecoveryTargetType] = useState("time");
+  const [recoveryTarget, setRecoveryTarget] = useState(
+    DateTime.utc().toFormat("yyyy-MM-dd HH:mm:ss"),
```

Comment:
Should this be ISO8601?

```suggestion
    DateTime.utc().toFormat("yyyy-MM-ddTHH:mm:ssZ"),
```